ccd-definitions/src/test/java/uk/gov/hmcts/reform/divorce/ccd/ccdcase/NoFaultDivorceTest.java

```diff
@@ -8,8 +8,8 @@
 
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyNoMoreInteractions;
-import static uk.gov.hmcts.reform.divorce.ccd.model.Constants.CASE_TYPE;
-import static uk.gov.hmcts.reform.divorce.ccd.model.Constants.JURISDICTION;
+import static uk.gov.hmcts.reform.divorce.ccd.ccdcase.NoFaultDivorce.CASE_TYPE;
```

Comment:
may be we should update package name `uk.gov.hmcts.reform.divorce.ccd.ccdcase.NoFaultDivorce.CASE_TYPE` to 
`uk.gov.hmcts.reform.divorce.ccd.case.NoFaultDivorce.CASE_TYPE`
`ccd.ccdcase` doesn't sound too great we can change this later not for this PR

Reply:
I had the same thought! Unfortunately case is a keyword. I might just move it to ccd at the root.